src/components/EditorBottomBar.tsx

```diff
@@ -1,15 +1,32 @@
 import "./EditorBottomBar.css";
 import { FormattedMessage } from "react-intl";
+import { useAppServices } from "./AppServicesContext";
 
-interface EditorBottomBarProps {}
+interface EditorBottomBarProps {
+  nextSrc: string;
+}
+
+export const EditorBottomBar = ({
+  nextSrc,
+  ...otherProps
+}: EditorBottomBarProps) => {
+  const {
+    window: { location },
+  } = useAppServices();
+  const onContinue = (): any => {
```

Comment:
No usaría onContinue como nombre porque esta función no esta escuchando un evento sugiero onClick, goToNext ó goToNextUrl
```suggestion
  const goToNextUrl = (): any => {
```